engine/dlib/src/dlib/socket.cpp

```diff
@@ -119,7 +119,7 @@ namespace dmSocket
 #else
     Result GetHostByNameT(const char* name, Address* address, uint64_t timeout, int* cancelflag, bool ipv4, bool ipv6)
     {
-        const uint32_t THREAD_STACK_SIZE = 0x20000;
+        const uint32_t THREAD_STACK_SIZE = 256*1024;
```

Comment:
The CI+Asan test gives us a stack overflow from this function/thread.
So let's double the size and see if it works.
https://github.com/defold/defold/actions/runs/8625372374/job/23641868009